common/src/main/resources/assets/voicechat/lang/en_us.json

```diff
@@ -1,217 +1,212 @@
+  "message.voicechat.group_members": "Group members:",
+  "message.voicechat.no_group_members": "Empty group",
+  "message.voicechat.join_create_group": "Join or Create a Group",
+  "message.voicechat.group_name": "Group Name",
+  "message.voicechat.leave_group": "Press to leave the group.",
   "message.voicechat.not_in_group": "You are not in a group",
   "message.voicechat.invite": "%s invited you to the group %s. %s",
   "message.voicechat.accept_invite": "ACCEPT",
-  "message.voicechat.accept_invite.hover": "Click to accept invitation",
-  "message.voicechat.join_successful": "Successfully joined %s",
+  "message.voicechat.accept_invite.hover": "Click to accept the invitation",
+  "message.voicechat.join_successful": "Successfully joined the group %s",
   "message.voicechat.leave_successful": "Successfully left the group",
   "message.voicechat.groups_disabled": "Groups are disabled on this server",
   "message.voicechat.recording_disabled": "Recording is disabled on this server",
   "message.voicechat.invite_successful": "Successfully invited %s",
   "message.voicechat.processing_recording_session": "Processing recording session...",
   "message.voicechat.processing_progress": "Processing %s%%",
-  "message.voicechat.save_session": "Saved session to '%s'",
+  "message.voicechat.save_session": "Saved session to %s",
   "message.voicechat.save_session_failed": "Failed to save recording session: %s",
   "message.voicechat.recording_started": "Started Recording",
   "message.voicechat.recording_stopped": "Stopped Recording",
-  "message.voicechat.open_folder": "Open Folder",
-  "message.voicechat.storage_size": "Expected Storage Size %s",
-  "message.voicechat.saved_debug_report": "Saved voice chat debug report to %s",
-  "message.voicechat.open": "Open",
-  "message.voicechat.saved_debug_report_failed": "Failed to save voice chat debug report: %s",
+  "message.voicechat.open_folder": "Open folder",
+  "message.voicechat.storage_size": "Estimated space usage: %s.",
   "message.voicechat.microphone_unavailable": "Microphone unavailable",
-  "message.voicechat.speaker_unavailable": "Speaker unavailable",
+  "message.voicechat.speaker_unavailable": "Speakers unavailable",
   "message.voicechat.playback_unavailable": "Playback unavailable",
-  "message.voicechat.denoiser": "Noise suppression: %s",
-  "message.voicechat.enabled": "Enabled",
-  "message.voicechat.disabled": "Disabled",
+  "message.voicechat.denoiser": "Noise Suppression: %s",
+  "message.voicechat.enabled": "ON",
+  "message.voicechat.disabled": "OFF",
   "message.voicechat.group_type_title": "%s (%s)",
   "message.voicechat.group_title": "%s",
-  "message.voicechat.group_does_not_exist": "This group does not exist",
-  "message.voicechat.group_name_not_unique": "Group name is not unique",
+  "message.voicechat.group_does_not_exist": "Such a group does not exist",
+  "message.voicechat.group_name_not_unique": "Group name is ambiguous",
```

Comment:
This word might confuse people.

Reply:
You're absolutely right. However, this is a perfect word for this type of situation where two teams have the same name, and the command to join a team can't figure out which one you mean. It's a textbook example of ambiguity.

The previous form, in my opinion, did a much worse job of showing the cause of the problem. It should rather be displayed earlier, during the team creation process, to ensure that team names are unique right from the start. This would also make it easier to find the team you're looking for in the voice chat menu. (Which is a pretty good feature request, BTW)

I wouldn't worry too much about it, though, as I bet that at least 90% of all players use the graphical interface to join groups anyway, especially since it offers the same functionality as commands and even more. Furthermore, commands, by their nature, are aimed at more tech-savvy players, and they shouldn't have any trouble understanding error messages like this one.

Reply:
Yeah thats fair.